migrations/env.py

```diff
@@ -93,19 +105,13 @@ def process_revision_directives(context, revision, directives):
                 directives[:] = []
                 logger.info("No changes in schema detected.")
 
-    connectable = engine_from_config(
-        config.get_section(config.config_ini_section),
-        prefix="sqlalchemy.",
-        poolclass=pool.NullPool,
-    )
+    connectable = get_engine()
 
     with connectable.connect() as connection:
         context.configure(
             connection=connection,
-            target_metadata=target_metadata,
+            target_metadata=get_metadata(),
             process_revision_directives=process_revision_directives,
-            render_as_batch=True,
-            compare_type=True,
```

Comment:
compare_type is important to catch type changes of a column. Is this true by default now?